int8 conversation_type # Type of coversation, e.g. "text", "speech"
int8 conversation_id # ID of coversation, 0 is reserved for speech
string text
